josue/js/app.js

```diff
@@ -8,74 +8,104 @@ const registerFieldsWrapper = document.querySelector("#register-fields");
 const boardWrapper = document.querySelector(".account__board-wrapper");
 const userDashboard = document.querySelector(".account__board-user");
 const welcomeSplash = document.querySelector('.account__splash-img');
-const accountSection =document.querySelector(".account");
-const fields = ['name','phone'];
-const registerFieldsIds = ['email','password','password_confirmation'];
+const accountSection = document.querySelector(".account");
+const fields = ['name', 'phone'];
+const registerFieldsIds = ['email', 'password', 'password_confirmation'];
+const registrationFormWrapper = document.querySelector('.account__left-wrapper');
+const USERDASHBOARDBTNS = {
+    DEPOSIT: document.querySelector("#deposit"),
+    WITHDRAW: document.querySelector("#withdraw"),
+    TRANSACTIONS: document.querySelector("#transactions"),
+}
+let USER = undefined;
```

Comment:
Only constants should be in upper case + snake case. LET is for variables that are reassigned multiple times and they should be in lower case + camelCase